site/docs/get-started/quickstart.md

```diff
@@ -117,9 +117,13 @@ Fybrik assumes the existence of a data catalog that contains the metadata and co
 
 In this guide we use the OpenMetadata data catalog.
 
-To use OpenMetadata, you can either use an existing deployment, or run the following commands to deploy OpenMetadata in kubernetes:
+To use OpenMetadata, you can either use an existing deployment, or run the following commands to deploy OpenMetadata in kubernetes.
 
-=== "Kubernetes"
+*Note:* OpenMetadata deployment requires a cluster storage provisioner has persistent volumes capability of ReadWriteMany Access Mode.
```

Comment:
Suggestion:
*Note:* OpenMetadata deployment requires a cluster storage provisioner **that** has **PersistentVolume** capability of ReadWriteMany Access Mode.